.azure-devops/live-share-ci.yml

```diff
@@ -0,0 +1,85 @@
+# https://domoreexp.visualstudio.com/Teamspace/_build?definitionId=6457
+
+resources:
+  repositories:
+  - repository: SkypeSpaces-Infra
+    type: git
+    name: SkypeSpaces-Infra
+  - repository: MsTeamsServices-Tools
+    type: git
+    name: MsTeamsServices-Tools
+
+variables:
+- template: variables/variables.yml
+- name: esLintExclusionFilePath
+  value: "$(Build.SourcesDirectory)/.eslintignore"
+- name: esLintTargets
+  value: ''
+
+parameters:
+- name: version
+  type: string
+  default: "0.1.0"
```

Comment:
>"0.1.0" [](http://example.com/codeflow?start=11&length=7)

how does this get set?

Reply:
when running ADO pipeline

![image](https://user-images.githubusercontent.com/5934356/190214874-1b86e41b-6e24-41e7-bf5a-55d7f8a7ca34.png)